src/powerquery-language-services/inspection/scope/scopeUtils.ts

```diff
@@ -38,3 +52,68 @@ export function isSectionMember(maybeValue: TScopeItem | undefined): maybeValue
 export function isUndefined(maybeValue: TScopeItem | undefined): maybeValue is UndefinedScopeItem {
     return maybeValue?.kind === ScopeItemKind.Undefined;
 }
+
+export function findScopeItemByLiteral(
+    nodeScope: NodeScope | undefined,
+    literalString: string,
+): TScopeItem | undefined {
+    // eslint-disable-next-line no-nested-ternary
+    return nodeScope?.has(`@${literalString}`)
+        ? nodeScope.get(`@${literalString}`)
+        : nodeScope?.has(literalString)
+        ? nodeScope.get(literalString)
+        : undefined;
+}
+
+export function findTheCreatorIdentifierOfOneScopeItem(
+    scopeItem: TScopeItem | undefined,
+): Ast.Identifier | Ast.GeneralizedIdentifier | undefined {
+    if (!scopeItem) return undefined;
```

Comment:
Hmm I should've had a lint rule here to always prefer braces. I'll push out a change to this PR which adds the rule.